app/services/unit_activity_executions_exporter.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+class UnitActivityExecutionsExporter
+  attr_reader :activity_executions
+
+  HEADERS = %w[
+    id
+    activity_id
+    activity_name
+    activity_category
+    starts_at
+    ends_at
+    spot_id
+    spot_name
+    field_id
+    field_name
+    languages
+    max_headcount
+    headcount
+    unitscount
+    units
+    transport
+    transport_ids
+  ].freeze
+
+  def initialize(activity_executions)
```

Comment:
Würde hier direkt unit_activity_executions als Parameter nehmen, sonst passt es nicht zu diesem Exporternamen

Reply:
Changed the export to be one dimensional